.github/copilot-instructions.md

```diff
@@ -0,0 +1,238 @@
+# GitHub Copilot Instructions for RocketPy
+
+This file provides instructions for GitHub Copilot when working on the RocketPy codebase.
+These guidelines help ensure consistency with the project's coding standards and development practices.
+
+## Project Overview
+
+RocketPy is a Python library for 6-DOF rocket trajectory simulation.
+It's designed for high-power rocketry applications with focus on accuracy, performance, and ease of use.
+
+## Coding Standards
+
+### Naming Conventions
+- **Use `snake_case` for all new code** - variables, functions, methods, and modules
+- **Use descriptive names** - prefer `angle_of_attack` over `a` or `alpha`
+- **Class names use PascalCase** - e.g., `SolidMotor`, `Environment`, `Flight`
+- **Constants use UPPER_SNAKE_CASE** - e.g., `DEFAULT_GRAVITY`, `EARTH_RADIUS`
+
+### Code Style
+- Follow **PEP 8** guidelines
+- Use **Black formatter** for code formatting
+- Line length: **88 characters** (Black's default)
```

Comment:
I understand that `ruff` and `black` code formatting is very similar, but shouldn't we update this in favor of `ruff`? The same applies to the `isort` line.